recipes/disjoint-set/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "disjoint-set" %}
+{% set version = "0.7.1" %}
+{% set blake2_sha256 = "41a0caecf7d275fe5146c6afb80333c3e233388171432bf3bdf1e3633e990c0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/{{ blake2_sha256[0:2] }}/{{ blake2_sha256[2:4] }}/{{ blake2_sha256[4:] }}/disjoint_set-{{ version }}.tar.gz
+  sha256: 4012d7d96b049872b6d431fe1f06484c0736bfd5d43bfcb8106747ce70ce5027
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry >=1.1.0
+    - python >=3.7
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - disjoint_set
+
+about:
+  home: https://pypi.org/project/disjoint-set/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
Add a comment in the recipe that the license file is manually packaged. It would be good to raise an issue upstream to try to get the license in the source distribution.

Reply:
Added a comment and submitted an issue asking the license being included in the PyPi tarball.

Thank you very much for your recommendation!

Reply:
Thanks for doing that. If the license file is in the next release you should remove the manually packaged copy and delete the comment (it's just there as a visual reminder).

Reply:
Will do